### ISSUE TITLE
Fire and Forget downloading

### ISSUE THREAD

**Issue by SonarrBot** [Sonarr/Sonarr#329]
Support for setups that would like all their files in one folder (regardless of series) and use another mechanism (outside of the sorting within drone) to achieve it.

Possible Actions:

- Do nothing
- Immediately ignore episodes after they are queued (advanced option), but that would prevent failed downloads from being re-added if it was later reposted.
- API endpoint + PP script to send success from SAB to drone with a flag to say do not sort
- Add a flag to settings for F&F and check history on search if thats been set, if the last event it grabbed then don't grab again unless its a better quality and the cutoff hasn't been met for the grab
- Use CDH to mark the episode as downloaded (new status?), but do not import it.

**Comment by the0neWhoKnocks** [comment]
@markus101 - thanks for pointing me to this. So I take it there's nothing available that will allow me to do what I mentioned? I find it hard to believe that no one else has run across this issue in the almost three years since this ticket's been opened.

**Comment by markus101** [comment]
They have, there are a few threads on the forums (where we do support). The number is far less than the people that are using Sonarr as a complete system and this feature is not a high priority item for us.

**Comment by the0neWhoKnocks** [comment]
@markus101 - As I mentioned in my previous ticket, I've been through the forums and haven't found anything. If you could point me in the right direction, or better yet, if you have links to the posts, it would be greatly appreciated. Also, thanks for taking the time.

**Comment by markus101** [comment]
Its not a scenario we support so the forum posts are just going to link you back here (or to Trello when we were using that). Searching for this URL shows one thread, https://github.com/Sonarr/Sonarr/issues/329

**Comment by wipeoutstu** [comment]
Hi, just to confirm, is saving downloads to the same folder regardless of series name not something that is being looked into?
I always watch stuff as its available, and having to remember whats been on, then run through all the different folders is more hassle than renaming and organising the folder structure myself?
Don't get me wrong, i am massively impressed with Sonarr's capabilities, it just seems a shame to not have this simple option (you can already set the root folder and change individual series folders.)
Thanks

**Comment by the0neWhoKnocks** [comment]
@wipeoutstu - That was the impression I got. I ended up having to create a Python Cron job that my server runs once every morning. It uses the Sonarr API to get a list of `wanted` episodes within a 3 day window, then looks at my `unwatched` directory (where I store new stuff). If it finds a file that matches a `wanted` item, it makes an API call to mark the episodes `monitored` property to `false`.
